polling/frequent/README.md

```diff
@@ -0,0 +1,20 @@
+# Frequent polling
+
+This sample shows how we can implement frequent polling (1 second or faster) inside our activity. The implementation is a loop that polls our service and then sleeps for the poll interval (1 second in the sample).
```

Comment:
> is a loop that polls our service and then sleeps for the poll interval (1 second in the sample).

That does match the code I am seeing. Notice how the other frequent polling activity impls in other langs properly loop